docs/designs/conceptual-localization.md

```diff
@@ -0,0 +1,253 @@
+# Conceptual Localization(Loc) Support
+
+## Description
+
+DocFX build supports localization contents, there are a few features need to be supported for localization contents:
+  - Partial contents:
+    - The localization contents are often a part of source content.
+    - There maybe extra localization content which doesn't exist in source repo.
+  - Delay localization, which means the version of loc content are usually behind of source content version.
+  - Localization contents are not required to be stored in the same repo of source content, they can be anywhere.
+  - Localization publishing may have a few specific requirements which are different with source publishing:
+    - Most of the localization configuration would be same with source configuration, a few are different, set based on scenarios, so we don't know the exact set of different configurations.
+    - They may applies different configuration per branches.
+## Design
+
+- Treats localization content as the replacing content of source, they are not independent repo for publishing but only stores the corresponding loc content.
+- Localization publishing mixes the loc content and source content, loc content has higher priority to replace the source content.
+- Localization publishing uses source configuration and localization **overwrite** configuration
+
+## Workflow
+
+![Localization Publishing And Build](images/loc_build_publish.PNG)
+
+### Loc Content as Assets of Source Repo 
+
+  - No more specified fallback features, there is always mixed (Loc + Source) full set content for build/publishing  
+  - All features are equal to Source + Loc, less Loc repo specified features  
```

Comment:
There is few exception, e.g. dynamic scope is not supported in loc. Document this here or somewhere else?